test/test_sorted_containers.jl

```diff
@@ -1,18 +1,24 @@
-import Base.Ordering
-import Base.Forward
-import Base.Reverse
-import DataStructures.eq
+using Base: Ordering, Forward, Reverse, ForwardOrdering,
+            ReverseOrdering
```

Comment:
this fits on one line
```suggestion
using Base: Ordering, Forward, Reverse, ForwardOrdering, ReverseOrdering
```

Reply:
Done